server/project/models/creds.py

```diff
@@ -0,0 +1,20 @@
+from project import db
+
+
+class Credential(db.Model):
+    __tablename__ = "credentials"
+
+    id = db.Column(db.Integer, primary_key=True)
+    access_token = db.Column(db.String(2048), unique=True, nullable=False)
+    refresh_token = db.Column(db.String(2048), unique=True)
```

Comment:
Can `nullable=False` be added here?

Reply:
Currrently I'm not getting the refresh token from the client, still need to figure that out. The option I added to the  `GoogleLogin` component to give me the token should be correct, but not working.